conda_lsst/recipe_maker.py

```diff
@@ -84,7 +84,7 @@ def gen_conda_package(self, product, sha, eups_version, giturl, eups_deps, eups_
 		# the lsst-product-configs package as well, as that's where the .table
 		# and .cfg files are.
 		if depends_on_internal:
-			product_configs = 'lsst-product-configs ==0.0.1'
+			product_configs = 'lsst-product-configs ==1.0.1925'
```

Comment:
I was intending to ask your about this but thoughts about this but decided maybe I should wait until after the JTM.  

My first impression was that it would be better to set it in the config.yaml, in the hopes that some day multiple different bNNNNs could be supported with the same core code and just changing the config.yaml (and possibly other `./etc` data).  Today, I'm wondering if that is bit premature to worry about and it would be more convenient to pickup the version automatically.  Hypothetically, would that interact badly with trying to build ontop of an existing conda channel with multiple version of `lsst-product-config`?